File: chroma-league/src/main/java/net/booone/chromaleague/hud/parts/resource/NoResourceBar.java
package net.booone.chromaleague.hud.parts.resource;

import net.booone.razersdk.animation.AnimatedFrame;
import net.booone.razersdk.animation.Frame;
import net.booone.razersdk.animation.SimpleFrame;

public class NoResourceBar extends AnimatedFrame {

    @Override
    public Frame getFrame() {
        addAnimationFrame(new SimpleFrame());
        return super.getFrame();
    }
}

File: src/hud/parts/resource/NoResourceBar.ts
import {AnimatedFrame} from '../../../razer-sdk/animation/AnimatedFrame.js';
import {Frame} from '../../../razer-sdk/animation/Frame.js';
import {SimpleFrame} from '../../../razer-sdk/animation/SimpleFrame.js';

export class NoResourceBar extends AnimatedFrame {
    pollFrame(): Frame {
        this.addAnimationFrame(new SimpleFrame());
        return super.pollFrame();
    }
}
